hoot-core-test/src/test/cpp/hoot/core/algorithms/splitter/OsmMapSplitterTest.cpp

```diff
@@ -98,6 +98,6 @@ class OsmMapSplitterTest : public HootTestFixture
   }
 };
 
-CPPUNIT_TEST_SUITE_NAMED_REGISTRATION(OsmMapSplitterTest, "quick");
+CPPUNIT_TEST_SUITE_NAMED_REGISTRATION(OsmMapSplitterTest, "slow");
 
```

Comment:
Did some test category auditing throughout. So, a lot of changes related to that.